packages/core/src/lib/base.ts

```diff
@@ -110,30 +104,33 @@ export abstract class FrontendBase<B extends Backend, O extends Options>
   /**
    * @inheritDoc
    */
-  public async captureException(exception: any): Promise<void> {
-    const event = await this.getBackend().eventFromException(exception);
-    await this.captureEvent(event);
+  public async captureException(exception: any, scope: Scope): Promise<void> {
```

Comment:
Just a matter of preference, but since `scope` is the one consistent (and non-optional) argument to all functions, I'd make it the first.

Reply:
Or alternatively we make the scope optional if there is a reason to do it.

Reply:
We can make it optional and fallback to the `initialScope`